kbchat/test_config.example.yaml

```
@@ -20,7 +20,10 @@ config:
   teams:
     acme:
       # A real team that you add your alice1, alice2, and bob1 all into
-      teamname: "acme"
+      name: "acme"
+      # The channel to use
+      topicname: "mysupercoolchannel"
     alicesPlayground:
       # a team with alice in it AS ADMIN, but bob or charlie not in team
```

```suggestion
      # a team where alice is an admin
```
^ can whatever test uses this just set it up correctly? it should assert that it's correct before running anyway.